File: excel-sheet-column-title/excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        
        std::string title = "";
        
        while (columnNumber != 0) {
            
            columnNumber -= 1;
            
            char ch = 'A' + (columnNumber % 26);
            
            title.insert(title.begin(), ch);
            
            columnNumber /= 26;
        }
            
        return title;
    }
};

File: excel-sheet-column-title/excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        
        StringBuilder title = new StringBuilder();
        
        while (columnNumber != 0) {
            
            columnNumber -= 1;
            
            char ch = (char)('A' + (columnNumber % 26));
            
            title.append(ch);
            
            columnNumber /= 26;
        }
        
        title.reverse();
            
        return title.toString();
    }
}